lib/buildpackutil.py

```diff
@@ -7,6 +7,7 @@
 import sys
 from distutils.util import strtobool
 from urllib.parse import parse_qs
+from m2ee.version import MXVersion
```

Comment:
Please follow pep8 and put this in a separate group of imports for first-party modules (insert 1 line above this)

Reply:
bump